{"name": "故障告警类型", "status": "passed", "attachments": [{"name": "log", "source": "ad52c04d-16c7-42d4-a371-f7d8a350827c-attachment.txt", "type": "text/plain"}], "start": 1692236215667, "stop": 1692236215713, "uuid": "8f0da5b2-3d8b-4969-9d1e-1247d54d0d96", "historyId": "41246cfd2236c6b7ac7746b23ea9928d", "testCaseId": "3e54e5e903ce4b367ac549987cbede5f", "fullName": "opsPC.test_alarm#test_alarm_fault_type_001", "labels": [{"name": "feature", "value": "运维管理平台"}, {"name": "story", "value": "应急处置"}, {"name": "parentSuite", "value": "opsPC"}, {"name": "suite", "value": "test_alarm"}, {"name": "host", "value": "cale"}, {"name": "thread", "value": "13776-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "opsPC.test_alarm"}]}